ec_api/apps/users/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+__all__ = ["RegisterForm", "LoginForm"]
+
+
+class RegisterForm(forms.ModelForm):
+    """
+    Registration form for a User
+    """
+
+    class Meta:
+        model = User
+        fields = ["email"]
+        error_messages = {
+            "email": {
+                "unique": "User with this email has already registered. Do you need to log in?"
```

Comment:
Is this ever used?

Reply:
It is if you try to register with an email that is already registered... 
<img width="1552" alt="Screenshot 2021-03-02 at 10 49 53" src="https://user-images.githubusercontent.com/15347726/109637688-08786580-7b45-11eb-84ed-12dccf4eafcc.png">

Whether or not we want to show it is another question though. I know we don't for login (will always just display "check your email") but wasn't sure what's best practice for registration

Reply:
But in this situation we can just send the token? As in, we don't need to have different forms for each action, when they're always the same?